h2o-r/tests/testdir_algos/gam/runit_GH_16149_gam_row_error.R

```diff
@@ -0,0 +1,56 @@
+setwd(normalizePath(dirname(R.utils::commandArgs(asValues=TRUE)$"f")))
+source("../../../scripts/h2o-r-test-setup.R")
+
+library(data.table)
+library(dplyr)
+library(h2o) # or load your h2o in a different way
+library(ggplot2)
```

Comment:
IMHO those three are not necessary to be here to run the test

Reply:
only data.table is necessary to define

Reply:
Removed other libraries.